frontend/ui/uimanager.lua

```diff
@@ -1109,7 +1109,7 @@ function UIManager:suspend()
     if Device:isCervantes() or Device:isKobo() or Device:isSDL() or Device:isSonyPRSTUX() then
         self.event_handlers["Suspend"]()
     elseif Device:isKindle() then
-        self.event_handlers["IntoSS"]()
+        Device.powerd:toggleSuspend(1)
```

Comment:
Is the idea that this power button press emulation will trigger an `IntoSS` event when that's handled in `frontend/device/input`?

Reply:
Tbh I'm not certain, the idea is that power button suspends the device properly, so instead of creating such fixes, it directly emulates "power button press", thus triggering events related to this. It works normally and I don't think there could be any problems related to this. It is also what @NiLuJe suggested here https://github.com/koreader/koreader/issues/4823#issuecomment-474962797
It seems to work pretty well too.

Reply:
Yep, it'll fire the original event that IntoSS responds to.